tests/input_set.py

```diff
@@ -135,6 +135,116 @@ def create_geometry(self):
         super(PinCellInput, self).create_geometry()
 
 
+class GridInput(InputSet):
+    """Sets up a simple grid problem using explicit plane"""
+
+    def create_materials(self):
+        """Instantiate C5G7 Materials."""
+        self.materials = \
+            openmoc.materialize.load_from_hdf5(filename='c5g7-mgxs.h5',
+                                               directory='../../sample-input/')
+
+    def create_geometry(self):
+        """Instantiate a 3x3 grid Geometry."""
+
+        # Create the planes that bound the cell and geometry
+        xplanes = [None] * 4
+        yplanes = [None] * 4
+        for i in range(4):
+            val = -3.0 + 6.0/3 * i
+            xplanes[i] = openmoc.XPlane(x=val, name='xplane' + str(i))
+            yplanes[i] = openmoc.YPlane(y=val, name='yplane' + str(i))
+
+        xplanes[0].setBoundaryType(openmoc.REFLECTIVE)
+        xplanes[-1].setBoundaryType(openmoc.REFLECTIVE)
+        yplanes[0].setBoundaryType(openmoc.REFLECTIVE)
+        yplanes[-1].setBoundaryType(openmoc.REFLECTIVE)
+
+        # Create the cells and set the central cell to UO2, the rest as water
+        cells = [[None]*3 for i in range(3)]
+        for i in range(3):
+            for j in range(3):
+                cells[i][j] = openmoc.Cell(name='Cell ' + str(3*i+j))
+                if i == 1 and j == 1:
+                    cells[i][j].setFill(self.materials['UO2'])
+                else:
+                    cells[i][j].setFill(self.materials['Water'])
+                cells[i][j].addSurface(halfspace=+1, surface=xplanes[i])
+                cells[i][j].addSurface(halfspace=-1, surface=xplanes[i+1])
+                cells[i][j].addSurface(halfspace=+1, surface=yplanes[j])
+                cells[i][j].addSurface(halfspace=-1, surface=yplanes[j+1])
+
+        # Add the cells to the universe
+        root_universe = openmoc.Universe(name='root universe')
+        for i in range(3):
+            for j in range(3):
+                root_universe.addCell(cells[i][j])
+
+        self.geometry = openmoc.Geometry()
+        self.geometry.setRootUniverse(root_universe)
+
+        super(GridInput, self).create_geometry()
```

Comment:
What is the value of calling the parent class's `create_geometry()` method here?